src/index.ts

```diff
@@ -23,9 +23,15 @@ export const beautifier: Beautifier = {
       name: "ESLint",
       package: "eslint",
     },
+    {
+      type: DependencyType.Node,
+      name: "ESLint-plugin-React",
+      package: "eslint-plugin-react",
+    },
```

Comment:
Should be `optional`. cc @szeck87 from https://github.com/Unibeautify/beautifier-eslint/issues/87#issuecomment-417022796

Reply:
Yep, please add `optional: true`